cassandraprotocol/frame/encoder.go

```diff
@@ -101,6 +120,27 @@ func (c *Codec) encodeHeader(frame *Frame, bodyLength int, dest io.Writer) error
 	return nil
 }
 
+func (c *Codec) encodeRawHeader(rawHeader *RawHeader, bodyLength int, dest io.Writer) error {
```

Comment:
You don't need to pass `bodyLength` separately, it's already present in the `RawHeader` struct.

Reply:
done